reactor-core/src/test/java/reactor/core/publisher/MonoCollectTest.java

```diff
@@ -48,6 +51,19 @@
 
 	static final Logger LOGGER = Loggers.getLogger(MonoCollectListTest.class);
 
+	@RegisterExtension
+	AutoDisposingExtension afterTest = new AutoDisposingExtension();
+
+	private void lessVerboseLogs(Class<?> clazz) {
+		final ch.qos.logback.classic.Logger logbackLogger = (ch.qos.logback.classic.Logger) LoggerFactory.getLogger(clazz);
+
+		ch.qos.logback.classic.Level oldLevel = logbackLogger.getLevel();
+		logbackLogger.setLevel(ch.qos.logback.classic.Level.ERROR);
+		afterTest.autoDispose(() -> {
+			System.out.println("going back to level " + oldLevel);
```

Comment:
Do we need this `System.out`? If yes can we specify also the logger?

Reply:
yeah we could remove that part